api/manifest.proto

```diff
@@ -20,33 +20,14 @@ option go_package = "github.com/tetratelabs/getenvoy/api;api";
 message Manifest {
    string manifest_version = 1;
 
-   // Key is the flavor name
+   // Flavor is always "standard" for historical reasons
```

Comment:
Adding a note rather than break the proto.

Later, we can refactor the proto. I think the json source for this proto and where it is hosted may always be in our repo frankly, so we should be able to break compat once we figure out the stable shape that works with upstream hosted builds. In any case we can keep compat for now cc @musaprg